### PR TITLE
[fix-5003]: kto header

### PR DESCRIPTION
**Pull request by vdegraaf**
Ticket: [SIG-5003](https://gemeente-amsterdam.atlassian.net/browse/SIG-5003)

Add a topic to "Over iets anders" answer in the negative feedback kto form. 

Currently vs. New
Screenshot 2023-02-07 at 11 34 42

### CODE REVIEW

**Review by OscarBakker — APPROVED** [review]
goodgood

**Review by SireeKoolenWijkstra — CHANGES_REQUESTED** [review]
You also changed it for KTO positive. I don't think that's what should happen.

File: src/signals/incident/containers/KtoContainer/index.test.js [hunk 1]
```diff
@@ -111,6 +111,8 @@ describe('signals/incident/containers/KtoContainer', () => {
     await findByTestId('kto-form-container')
 
     expect(getByText(/nee, ik ben niet/i)).toBeInTheDocument()
+
```
Comment (SireeKoolenWijkstra) [inline src/signals/incident/containers/KtoContainer/index.test.js:114]: This doesn't exclude it from happening in the KTO positive flow.

**Comment by vdegraaf** [comment]
@SireeKoolenWijkstra the change is good for both positive and negative kto form. There was an issue with a double header in the positive flow, but that has been fixed by Jelle now. Check: 
Screenshot 2023-02-07 at 20 58 29

**Review by SireeKoolenWijkstra — APPROVED** [review]
Looks good to me

**Pull request merged**